ui/app/components/app/connected-accounts-list/connected-accounts-list-options/connected-accounts-list-options.component.js

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types'
+import React, { PureComponent } from 'react'
+import { Tooltip } from 'react-tippy'
+
+export default class ConnectedAccountsListOptions extends PureComponent {
+  static contextTypes = {
+    t: PropTypes.func.isRequired,
+  }
+
+  static propTypes = {
+    expanded: PropTypes.bool,
+    onClick: PropTypes.func.isRequired,
+    children: PropTypes.node.isRequired,
+  }
+
+  static defaultProps = {
+    expanded: false,
+  }
+
+  render () {
+    const { expanded, onClick, children } = this.props
+    const { t } = this.context
+
+    return (
+      <Tooltip
+        arrow={false}
+        animation="none"
+        animateFill={false}
+        transitionFlip={false}
+        hideDuration={0}
+        duration={0}
+        interactive
+        theme="none"
+        trigger="manual"
+        open={expanded}
```

Comment:
This options menu doesn't seem to close when you click outside of it. You need to click directly on the button again to toggle it closed.

Reply:
Updated with a caveat expressed below